Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -806,18 +807,27 @@ private async Task ProcessRequest(MessageRequest req, CancellationToken cancella
             TaskCompletionSource<IEnumerable<Message>> tcs = req.TaskCompletionSource;
             try
             {
-                await dealer.SendMultipartMessageAsync(
-                    message,
-                    timeout: req.Timeout,
-                    cancellationToken: cancellationToken
-                );
-
-                _logger.Debug(
-                    "Request {Message} {RequestId} sent to {Peer} with timeout {Timeout}.",
-                    req.Message,
-                    req.Id,
-                    req.Peer,
-                    req.Timeout);
+                if (dealer.TrySendMultipartMessage(message))
+                {
+                    _logger.Debug(
+                        "Request {Message} {RequestId} sent to {Peer} with timeout {Timeout}.",
```

Comment:
Seems like timeout is not needed anymore?

Reply:
It was to track overall timeout tied to a `RequestId`. I suppose the log message itself is confusing. Fixed in https://github.com/planetarium/libplanet/commit/debefd6a29941a2d771954f2444dd94c6443b77f.